src/structured-data/structured-data-tags.js

```diff
@@ -184,13 +186,52 @@ function generateWebSiteData(structuredData = {}, story = {}, publisherConfig =
   return getSchemaWebsite(structuredData.website);
 }
 
+function generateBreadcrumbListData(pageType = "", publisherConfig = {}, data = {}) {
+  const { "sketches-host": domain = "", sections = [] } = publisherConfig;
+  const breadcrumbList = getSchemaBreadcrumbList();
+  breadcrumbList.itemListElement.push(getSchemaListItem(1, "Home", domain));
+
+  function addCrumb(crumbsDataList = [], currentSection = {}) {
+    const parentSection = sections.find(section => section.id === currentSection["parent-id"]);
+
+    if(!parentSection) return crumbsDataList;
+
+    const { "section-url":sectionUrl = "", name = "" } = parentSection;
+    crumbsDataList.unshift({ sectionUrl, name });
+    return addCrumb(crumbsDataList, parentSection);
+  }
+
+  function getSectionPageCrumbs({ "section-url":sectionUrl = "", slug = "", name = "" } = {}) {
+    const crumbsDataList = [{ sectionUrl, name }];
+    const currentSection = sections.find(section => section.slug === slug);
+    addCrumb(crumbsDataList, currentSection);
+    return crumbsDataList.map(({sectionUrl = "", name = ""}, index) => getSchemaListItem(index + 2, name, sectionUrl));
+  }
+
+  function getStoryPageCrumbs({ headline = "", url = "", sections: storySections = [] } = {}) {
+    const storySectionSlug = get(storySections, [0, "slug"], "");
+    const currentSection = sections.find(section => section.slug === storySectionSlug);
+    const sectionCrumbsList = getSectionPageCrumbs(currentSection);
+    const position = sectionCrumbsList.length + 2;
+    sectionCrumbsList.push(getSchemaListItem(position, headline, url));
```

Comment:
again we should pass an array to getSchemaBreadcrumbList and that guy should worry about the position, and the schema.